pkg/kapp/cmd/appchange/list.go

```diff
@@ -98,3 +129,13 @@ func (t AppChangesTable) Print(ui ui.UI) {
 
 	ui.PrintTable(table)
 }
+
+func parseTime(input string, formats []string) (time.Time, error) {
```

Comment:
lets put this on `func (o *ListOptions)` to reduce scope where this function is available.

Reply:
Adopted.